pkg/resource/promtail-toggle/observability_bundle_configmap.go

```diff
@@ -19,16 +17,16 @@ type app struct {
 }
 
 // ObservabilityBundleConfigMapMeta returns metadata for the observability bundle user value configmap.
-func ObservabilityBundleConfigMapMeta(object client.Object) metav1.ObjectMeta {
+func ObservabilityBundleConfigMapMeta(lc loggedcluster.Interface) metav1.ObjectMeta {
 	return metav1.ObjectMeta{
-		Name:      fmt.Sprintf("%s-observability-bundle-user-values", object.GetName()),
-		Namespace: object.GetName(),
+		Name:      lc.GetAppName("observability-bundle-user-values"),
```

Comment:
This is not an app name. It is just a fixed name we use to create a configmap used a user value in the observability bundle app. Maybe we should have a fix name like `observability-bundle-logging-operator-user-values` ?

Reply:
Alright make sense to keep better names for uniqueness.

Or maybe `AppUserValueName` ? I have no strong opinion

Reply:
Fine with me then :)